completions/ssh.completion.sh

```diff
@@ -16,37 +16,42 @@ function _omb_completion_ssh {
 
   local IFS=$'\n'
 
-  # parse all defined hosts from .ssh/config
-  if [[ -r ~/.ssh/config ]]; then
-    local -a config_files=(~/.ssh/config)
-
-    # check if .ssh/config contains Include options
-    local -a include_patterns
-    _omb_util_split include_patterns "$(awk -F' ' '/^Include/{print $2}' ~/.ssh/config 2>/dev/null)" $'\n'
-    local i
-    for i in "${!include_patterns[@]}"; do
-      # relative or absolute path, if relative transforms to absolute
-      [[ ${include_patterns[i]} == /* ]] || include_patterns[i]=~/.ssh/${include_patterns[i]}
-    done
-
-    # interpret possible globbing
-    local -a include_files
-    _omb_util_glob_expand include_files '${include_patterns[*]}'
-    local include_file
-    for include_file in "${include_files[@]}";do
-      # parse all defined hosts from that file
-      [[ -s $include_file ]] && config_files+=("$include_file")
-    done
-
-    COMPREPLY+=($(compgen -W "$(awk '/^Host/ {for (i=2; i<=NF; i++) print $i}' "${config_files[@]}")" "${options[@]}"))
-  fi
+  for base_config_file in ~/.ssh/config /etc/ssh/ssh_config; do
+    # parse all defined hosts from config file
+    if [[ -r $base_config_file ]]; then
+      local basedir
+      basedir=$(dirname $base_config_file)
+      local -a config_files=("$base_config_file")
+
+      # check if config file contains Include options
+      local -a include_patterns
+      _omb_util_split include_patterns "$(awk -F' ' '/^Include/{print $2}' ${base_config_file} 2>/dev/null)" $'\n'
```

Comment:
```suggestion
      _omb_util_split include_patterns "$(awk -F' ' '/^Include/{print $2}' "${base_config_file}" 2>/dev/null)" $'\n'
```

The parameter expansion should be properly quoted.

Reply:
99a2f8c